content/en/blog/simplifying-islands-arch.md

```diff
@@ -0,0 +1,459 @@
+---
+title: Simplifying Islands Architecture
+date: 2024-10-27
+authors:
+  - reaper
+---
+ 
+# Islands
+
+## Intro
+
+This guide is a simple walkthrough to understand how island architechture works
+and being able to setup your own using tools you already have around you.
```

Comment:
Certainly feel free to drop a "This was previously posted at...." if you wanted to, we're happy to link to the original posting